make_deb

```diff
@@ -0,0 +1,165 @@
+              echo -n "Cleaning the build folder..."
+              rm -rf ${GIT_ROOT}/packages/build/*
+              echo "Done."
+          else
+              echo "No build folder, skipping cleanall"
+          fi
+          exit 0
+          ;;
+      help)
+          echo "${usage}"
+          exit 0
+          ;;
+      *)
+          echo "Unrecognized option: $i"
+         exit 1
+        ;;
+  esac
+done
+
+# If the architecture has not been defined, then the script is complete
+if [ "$(dpkg --print-architecture)" == "armhf" ]; then
+  echo "Building on arm architecture..."
+  architecture="armhf"
+elif [ "$(dpkg --print-architecture)" == "amd64" ]; then
+  echo "Building on amd64 architecture..."
+  architecture="x86_64"
+else
+  echo "The host architecture is not supported for this plugin!!"
+  exit 1
+fi
+
+# Get plugin version from VERSION
+plugin_version=`cat ${GIT_ROOT}/VERSION`
+# Get FogLAMP version dependency from foglamp_version file
+foglamp_version=`cat ${GIT_ROOT}/foglamp.version | tr -d ' ' | grep 'foglamp_version' | head -1 | sed -e 's/\(.*\)version\(.*\)/\2/g'`
+BUILD_ROOT="${GIT_ROOT}/packages/build"
+
+# Final package name
+package_name="foglamp-${plugin_type}-${plugin_name}-${plugin_version}-${architecture}"
+
+# Print the summary of findings
+echo "The package root directory is   : ${GIT_ROOT}"
+echo "The FogLAMP required version    : ${foglamp_version}"
+echo "The package will be built in    : ${BUILD_ROOT}"
+echo "The architecture is set as      : ${architecture}"
+echo "The package name is             : ${package_name}"
+echo
+
+# Create the package directory. If a directory with the same name exists,
+# it is copied with a version number.
+
+# First, create the BUILD_ROOT folder, if necessary
+if [ ! -L "${BUILD_ROOT}" -a ! -d "${BUILD_ROOT}" ]; then
+    mkdir -p "${BUILD_ROOT}"
+fi
+
+# Check if the default directory exists
+if [[ ! -d "${FOGLAMP_ROOT}" ]]; then
+    logger -p local0.err -t "foglamp.script.foglamp" "FogLAMP cannot be executed: ${FOGLAMP_ROOT} is not a valid directory."
+    echo "FogLAMP cannot be executed: ${FOGLAMP_ROOT} is not a valid directory."
+    echo "Create the enviroment variable FOGLAMP_ROOT before using FogLAMP."
+    echo "Specify the base directory for FogLAMP and set the variable with:"
+    echo "export FOGLAMP_ROOT=<basedir>"
+    exit 1
+fi
+
+# Check/set LD_LIBRARY_PATH
+libPathSet=0
+libdir=${FOGLAMP_ROOT}/lib; [ -d ${libdir} ] && LD_LIBRARY_PATH=$(echo $LD_LIBRARY_PATH | sed "s|${libdir}||g") && export LD_LIBRARY_PATH=${libdir}:${LD_LIBRARY_PATH} && libPathSet=1
+libdir=${FOGLAMP_ROOT}/cmake_build/C/lib; [ -d ${libdir} ] && LD_LIBRARY_PATH=$(echo $LD_LIBRARY_PATH | sed "s|${libdir}||g") && export LD_LIBRARY_PATH=${libdir}:${LD_LIBRARY_PATH} && libPathSet=1
+[ "$libPathSet" -eq "0" ] && echo "Unable to set/update LD_LIBRARY_PATH to include path of Foglamp shared libraries: check whether ${FOGLAMP_ROOT}/lib or ${FOGLAMP_ROOT}/cmake_build/C/lib exists" && exit 1
+
```

Comment:
Call the requirements.sh here

Reply:
@praveen-garg requirements.sh, with libcurl check, as in other plugins, is not needed here

Reply:
ah! yes, its a rule plugin only.

Please remove the requirements.sh from this branch as well.